contrib/ec2/setup-proxy-protocol.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash -e
+#
+# Usage: setup-proxy-protocol <cloudformation stack name>
+# <cloudformation stack name> -  should be the same as the name you ran provision-ec2-cluster.sh with. (default: deis)
+# 
+# The script can be executed at any time, as long as the CoreOS cluster is active.
+
+
+if [[ -z "$1" ]]; then
+	STACK_NAME=deis
+else
+	STACK_NAME=$1
+fi
+
+if ! aws cloudformation describe-stacks --stack-name ${STACK_NAME}; then
+	echo "Could not find cloudformation stack: ${STACK_NAME}"
+	exit 1
+fi
+
+LOAD_BALANCER_DNS=$(aws cloudformation describe-stacks --stack-name ${STACK_NAME} | grep -m 1 "OutputValue" | sed 's/ *//g' | cut -d ':' -f2 | cut -d '"' -f2)
+LOAD_BALANCER_NAME=$(echo -e ${LOAD_BALANCER_DNS} | cut -d '.' -f1 | cut -d '-' -f1-4)
+
+echo "Enabling Proxy Protocol"
+deisctl config router set proxyProtocol=1
+
+echo "Changing port 80 from HTTP -> TCP"
+aws elb delete-load-balancer-listeners \
+	--load-balancer-name ${LOAD_BALANCER_NAME} \
+	--load-balancer-ports 80
+
+aws elb create-load-balancer-listeners \
+	--load-balancer-name ${LOAD_BALANCER_NAME} \
+	--listeners Protocol=TCP,LoadBalancerPort=80,InstanceProtocol=TCP,InstancePort=80
+
+SSL_CERTIFICATE=$(aws elb describe-load-balancers --load-balancer-names ${LOAD_BALANCER_NAME} | grep -m 1 "SSLCertificateId" | sed 's/ *//g' | cut -d ':' -f2-10 | cut -d '"' -f2)
```

Comment:
Maybe, but I still need the SSL_CERTIFICATE in the next part.

It's not the prettiest solution. The script I created for the company uses [jq](https://github.com/stedolan/jq) but I didn't want to add extra dependencies to Deis.